src/libs/actions/PersonalDetails.ts

```diff
@@ -43,20 +43,19 @@ Onyx.connect({
 });
 
 /**
- * Returns the displayName for a user
+ * Creates a new displayName for a user based on passed personal details or login.
  */
-function getDisplayName(login: string, personalDetail: Pick<PersonalDetails, 'firstName' | 'lastName'> | null): string {
+function createDisplayName(login: string, personalDetails: Pick<PersonalDetails, 'firstName' | 'lastName'>): string {
```

Comment:
Just curious why we are changing the name. "create" implies to me that we're going to store this somewhere.

Reply:
Ahhh I see, ok that makes sense.